thirdparty/fetch-thirdparty-deps-windows.sh

```diff
@@ -8,30 +8,38 @@ download_dir="${0%/*}/download/windows"
 mkdir -p "${download_dir}"
 cd "${download_dir}"
 
-if [ ! -f SDL2.dll ]; then
+if [ ! -f SDL2-x86.dll -o ! -f SDL2-x64.dll ]; then
 	echo "Fetching SDL2 from libsdl.org"
 	curl -LOs https://www.libsdl.org/release/SDL2-2.0.5-win32-x86.zip
 	unzip SDL2-2.0.5-win32-x86.zip SDL2.dll
+	mv SDL2.dll SDL2-x86.dll
 	rm SDL2-2.0.5-win32-x86.zip
+	curl -LOs https://www.libsdl.org/release/SDL2-2.0.5-win32-x64.zip
+	unzip SDL2-2.0.5-win32-x86.zip SDL2.dll
```

Comment:
Small oversight. This should be:

```patch
-unzip SDL2-2.0.5-win32-x86.zip SDL2.dll
+unzip SDL2-2.0.5-win32-x64.zip SDL2.dll
```

Reply:
Thanks for spotting this - I was going blind trying to work out what was wrong.